3.4/x86/Dockerfile

```diff
@@ -1,15 +1,21 @@
 FROM i386/ubuntu:16.04
 LABEL MAINTAINER "Dashlorde <zhouyunlu0216@gmail.com>"
 
-VOLUME ["/var/moodledata", "/var/www/html"]
 EXPOSE 80 
 
-ADD ./start.sh /etc/apache2/start.sh
+ADD ./start.sh /etc/apache2/start.sh 
+
+ADD https://github.com/moodle/moodle/archive/v3.4.0.tar.gz /var/www/moodle-3.4.0.tgz
```

Comment:
use curl or wget better I think, I can't finish the add process in my mac, can you do it in your local?